controllers/release/adapter.go

```diff
@@ -44,6 +44,9 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+const defaultReleaseCatalogUrl = "https://github.com/konflux-ci/release-service-catalog.git"
```

Comment:
Not sure if this is the best place... for sure we don't have any other place to place them (metadata is for... metadata) but if we want to use this url in other places maybe it will cause problems (cycle dependencies). Won't be the case for tests though, so maybe it's fine. @johnbieren thoughts?

Reply:
I think it makes most sense in here https://github.com/konflux-ci/release-service/blob/main/api/v1alpha1/collectors.go as that is where we have all collector stuff (and these constants are only used for collectors for now). I also think a constant for the pathInRepo would make sense. I didn't check if it creates a cyclical dependency though

Reply:
moved to collectors.